war/src/main/less/base/style.less

```diff
@@ -513,6 +518,11 @@ table.bigtable.pane > tbody > tr > td:last-child {
   background-color: #f0f0f0;
   padding: 6px 4px;
   white-space: nowrap;
+  text-align: left;
+
+  &.minimum-width {
+    width: 1px;
```

Comment:
not sure how this would work. @EstherAF  was looking into tables lately and found some issues with this 1px width. Care to chime in?

Reply:
The problem I found doesn't affect here, I think, so this looks good to me.
My problem was setting a `width: 1%` in a cell when the table has a `width: auto`. In that situation, the table width `auto` is calculated as 100%, so the table doesn't fit the content anymore, but expands to 100%.